Actions/AL-Go-Helper.ps1

```diff
@@ -815,7 +824,9 @@ function CreateDevEnv {
         [Parameter(Mandatory=$true, ParameterSetName='local')]
         [pscredential] $credential,
         [Parameter(ParameterSetName='local')]
-        [string] $containerName = ""
+        [string] $containerName = "",
+        [string] $insiderSasToken = "",
+        [string] $LicenseFileUrl = ""

```

Comment:
does that mean that license should be available publicly as there is no authentication?